### PR TITLE
Busy Bottoms

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch7784]

File: src/components/FlowFooter.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+export enum FlowFooterConfig {
+  GENERAL = 'GENERAL',
+  METAMASK_SIGN = 'METAMASK_SIGN',
+  METAMASK_UNLOCK = 'METAMASK_UNLOCK',
+  GENERIC_WEB3 = 'GENERIC_WEB3',
+  LEDGER = 'LEDGER',
+  TREZOR = 'TREZOR',
+  WALLETCONNECT = 'WALLETCONNECT',
+  SUPPORT = 'SUPPORT'
+}
```
Comment (GuillaumeRx) [inline src/components/FlowFooter.tsx:30]: Can this go to config ?

File: src/components/FlowFooter.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  SUPPORT_EMAIL,
+  WALLETS_CONFIG
+} from '@config';
+import { translateRaw } from '@translations';
+import { TURL, WalletId } from '@types';
+import { getWeb3Config, openLink } from '@utils';
+
+import Box from './Box';
+import Icon from './Icon';
+import { Link } from './NewTypography';
```
Comment (GuillaumeRx) [inline src/components/FlowFooter.tsx:19]: from `@components`

File: src/components/FlowFooter.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    },
+    {
+      copy: 'FLOW_FOOTER_WALLETCONNECT_2',
+      link: getKBHelpArticle(KB_HELP_ARTICLE.HOW_TO_USE_WALLETCONNECT),
+      external: true
+    },
+    SUPPORT_LINK
+  ],
+  SUPPORT: [SUPPORT_LINK]
+};
```
Comment (GuillaumeRx) [inline src/components/FlowFooter.tsx:121]: This should go to config or in a constant file of you want to keep it near the component :)

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]

**Review by GuillaumeRx — APPROVED** [review]
lgtm

**Review by GuillaumeRx — DISMISSED** [review]

**Review by GuillaumeRx — DISMISSED** [review]

**Pull request merged**